modules/cnf-topology-aware-lifecycle-manager-about-cgu-crs.adoc

```diff
@@ -16,14 +16,26 @@ The {cgu-operator-first} builds the remediation plan from the `ClusterGroupUpgra
 * Actions to perform before and after the update
 * Update timing
 
-As {cgu-operator} works through remediation of the policies to the specified clusters, the `ClusterGroupUpgrade` CR can have the following states:
+You can control the start time of an update using the `enable` field in the `ClusterGroupUpgrade` CR.
+For example, if you have a scheduled maintenance window of four hours, you can prepare a `ClusterGroupUpgrade` CR with the `enable` field set to `false`.
 
-* `UpgradeNotStarted`
-* `UpgradeCannotStart`
-* `UpgradeNotComplete`
-* `UpgradeTimedOut`
-* `UpgradeCompleted`
-* `PrecachingRequired`
+You can set the timeout as follows:
+[source,yaml]
+----
+remediationStrategy:
+        maxConcurrency: 1
+        timeout: 240
+----
+
+You can use the `batchTimeoutAction` to determine what happens if an upgrade fails for a cluster.
+You can specify `continue` to skip the failing cluster and continue to upgrade other clusters, or `abort` to stop the upgrade for all clusters.
```

Comment:
The second part of this sentence feels misleading. We don't actually stop anything other the CGU itself, i.e. stop policy remediation on all clusters

Reply:
ok, I've reworded "....`abort` to stop policy remediation for all clusters."